### DISCUSSION TITLE
[Remove Request] -

### DISCUSSION THREAD

**Discussion by GivenupLol** [blocklistproject/Lists#925] [Q&A]
URL you wish to be removed:
0.0.0.0

Why you believe this to be a false positive:
Pi-Hole Believes this to be an invalid domain

List it is on:
Adobe

Other info you think we should know:

**Comment by gap579137** [comment]
Pihole auto strips the 0.0.0.0, what issue are you seeing that make you believe there is an issue? The "alt-version" doesn't have the leading 0.0.0.0 if you would like to use a list version that already has them stripped.
